### PR TITLE
Add job on backend for setting cow health for all users in a commons

### LINKED ISSUE #26: Add job on backend for setting cow health for all users in a commons
When the admin enters a value between 0 and 100 and clicks "set cow health", a job will start that sets the cow health of all users in that commons to that value.

### PR DESCRIPTION
**Pull request by TheRealJBomb**
In this PR, a job is added on the backend for setting the cow health for all users in a particular commons. The job expects a `commonsID` as well as a `health` value for it to work properly.

### CODE REVIEW

**Review by pconrad — APPROVED** [review]
LGTM.  Will test on staff-qa site before merging.

File: src/test/java/edu/ucsb/cs156/happiercows/jobs/SetCowHealthJobTests.java [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+                // Arrange
+
+                Job jobStarted = Job.builder().build();
+                JobContext ctx = new JobContext(null, jobStarted);
+
+                when(commonsRepository.findById(any())).thenReturn(Optional.empty());
+                
+                // Optional<Commons> commons = commonsRepository.findById(commonsID);
+                // Iterable<UserCommons> allUserCommons = userCommonsRepository.findByCommonsId(commons.get().getId());
```
Comment (pconrad) [inline src/test/java/edu/ucsb/cs156/happiercows/jobs/SetCowHealthJobTests.java]: can we remove this commented out code?

**Review by pconrad — DISMISSED** [review]
LGTM!  I've tested on staff-qa and it works great.

One nitpick: there is some commented out code.   We really try to avoid merging commented out code into the main branch.

If you can take care of that, this is mergeable!

File: src/test/java/edu/ucsb/cs156/happiercows/jobs/SetCowHealthJobTests.java (hunk 1 above)
Comment (TheRealJBomb) [reply src/test/java/edu/ucsb/cs156/happiercows/jobs/SetCowHealthJobTests.java]: removed!

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**